### PR TITLE
Max Chapter3/homework/7 and8

### PR DESCRIPTION
**Pull request by HellHunterMax**
LINQ Query Syntax is still hard to do for me but i do see the use of it.

### CODE REVIEW

File: Src/BootCamp.Chapter/LINQExtension.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+{
+    public static class LINQExtension
+    {
+        public static IEnumerable<T> SnapFingers<T>(this IEnumerable<T> source)
+        {
+            List<T> list = new List<T>();
+
+            int i = 0;
+
+            foreach (var element in source)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/LINQExtension.cs:13]: This is ``source.Count()``

Comment (Almantask) [reply Src/BootCamp.Chapter/LINQExtension.cs:13]: You should have used LINQ. Also there is no random factor what gets selected- always the first half.

Comment (Almantask) [reply Src/BootCamp.Chapter/LINQExtension.cs:13]: Can you implement a select using LINQ :) ?

**Review by Almantask — CHANGES_REQUESTED** [review]
Double check SnapFingers function

File: Src/BootCamp.Chapter/LINQExtension.cs (hunk 1 above)
Comment (HellHunterMax) [reply Src/BootCamp.Chapter/LINQExtension.cs:13]: woeps forgot about this.  missed the using LINQ thought it was weird i could not use Count().
can you please re review it for me thanks kai!

**Pull request closed without merging**